### DISCUSSION TITLE
Payload size limit exceeded

### DISCUSSION THREAD

**Discussion by jonandgon** [WalletConnect/walletconnect-monorepo#4100] [Developer Support]
NPM Versions: 
- `@walletconnect/web3wallet: ^1.10.0`
- `@walletconnect/core: ^2.11.0`
- `@walletconnect/sign-client: ^2.11.0`

When sending messages from wallet to dApp, the payload limit is constantly reached. The limit is roughly 71kb, but the 71kb limit isn't defined anywhere. This is the error message I get when the limit is reached:

`Payload size limit exceeded. Maximum size for tag 1109 is 98304`

This low limit is killing the UX for dApps and is our biggest pain point with WalletConnect. It would be nice if the limit was increased, or if there were an easy way to break up a large message and send the chunks.

Our wallet is for a private blockchain, so there's unfortunately no public data provider that can reduce the load on WalletConnect.

**Comment by jonandgon** [comment]
@glitch-txs can you take a look at this?

**Reply by glitch-txs** [reply]
Hi, could you open an issue in the monorepo?
